cmd/virtual-kubelet/root/root.go

```diff
@@ -173,7 +171,7 @@ func runRootCommand(ctx context.Context, s *provider.Store, c *Opts) error {
 		nodeProviderModule,
 		pNode,
 		client.CoreV1().Nodes(),
-		module.WithNodeEnableLeaseV1(leaseClient, nil),
+		module.WithNodeEnableLeaseV1(leaseClient, 0),
```

Comment:
```suggestion
		module.WithNodeEnableLeaseV1(leaseClient, module.DefaultLeaseDuration),
```

Nit. Just to explicit the meaning